plugins/woocommerce/includes/admin/settings/views/html-admin-page-shipping-zones.php

```diff
@@ -65,6 +65,11 @@
 
 </table>
 
+<?php
+// phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
+do_action( 'woocommerce_shipping_zones_after_zones_table' );
```

Comment:
As I understood, the intent of this hook is that you would like to display the zones table?

Reply:
The intent is to have a hook in which to be able to inject extra styling per shipping method. In my specific case, I'll be adding an icon.